### PR TITLE
Added test cases for Map:addHandler method

### LINKED ISSUE #8372: Implement test for 'addHandler' method of Map

### PR DESCRIPTION
**Pull request by shtelzerartem**
Resolve #8372
Added 3 test cases for Map:addHandler method

### CODE REVIEW

File: spec/suites/map/MapSpec.js [hunk 1]
```diff
@@ -611,6 +611,62 @@ describe("Map", function () {
...
+
+				handleClick: callback
+			});
+		}
+
+		it("checking enabled method", function () {
+			L.ClickHandler = getHandler();
+			map.addHandler('clickHandler', L.ClickHandler);
+
+			map.clickHandler.enable();
```
Comment (Falke-Design) [inline spec/suites/map/MapSpec.js:633]: Let us add a check, if it's disabled by default
```suggestion
			map.addHandler('clickHandler', L.ClickHandler);

			expect(map.clickHandler.enabled()).to.eql(false);

			map.clickHandler.enable();
```

File: spec/suites/map/MapSpec.js [hunk 2]
```diff
@@ -611,6 +611,62 @@ describe("Map", function () {
...
+			expect(map.clickHandler.enabled()).to.eql(true);
+
+			map.clickHandler.disable();
+			expect(map.clickHandler.enabled()).to.eql(false);
+		});
+
+		it("automatically enabled, when has a property named the same as the handler", function () {
+			container = createContainer();
+			map = L.map(container, {clickHandler: true});
```
Comment (Falke-Design) [inline spec/suites/map/MapSpec.js:642]: Don't create a new container. Remove the old map and use the same container
```suggestion
			map.remove();
			map = L.map(container, {clickHandler: true});
```

**Review by Falke-Design — CHANGES_REQUESTED** [review]
Very nice, thank you. Only a few small things

File: spec/suites/map/MapSpec.js (hunk 2 above)
Comment (shtelzerartem) [reply spec/suites/map/MapSpec.js:642]: Ahh, yeah, makes sense. Fixing.

**Review by Falke-Design — APPROVED** [review]
Very nice, thank you

**Pull request merged**